### DISCUSSION TITLE
Welcome to DirectoryOpus-DFIRConfig Discussions!

### DISCUSSION THREAD

**Discussion by AndrewRathbun** [AndrewRathbun/DirectoryOpus-DFIRConfig#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StrackRX** [comment]
All the pictures you showed are invalid. I hope they can be fixed.

**Comment by StrackRX** [comment]
![error](https://github.com/user-attachments/assets/d73326f6-560c-43ac-a355-3089b1d65990)

**Reply by AndrewRathbun** [reply]
This is fixed, sorry about that, and thank you for bringing it to my attention! Fixed here: https://github.com/AndrewRathbun/DirectoryOpus-DFIRConfig/issues/5
